content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -157,3 +157,36 @@ Only one of the load balancing options should be configured, as they are
 mutually exclusive. Attempting to configure both may lead to unexpected
 behavior.
 {{< /warning >}}
+
+{{< tip >}}
+In a single multi-zone cluster environment, locality load balancing can also be configured for failover to a different zone within the same cluster. To test it, you will need to create a cluster with multiple worker zones and deploy istiod instance and the app to each zone. The following example shows how to set up a kind cluster with multiple worker zones. The remaining steps are identical to this multicluster example.
+
+Generate yaml to define a cluster with 1 control-plane and 3 worker nodes:
+
+{{< text bash >}}
+$ cat <<EOF > kind-cluster-config.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+  - role: worker
+  - role: worker
+  - role: worker
+EOF
+{{< /text >}}
+
+Create a kind cluster with yaml generated from above:
+
+{{< text bash >}}
+$ kind create cluster --config kind-cluster-config.yaml
+{{< /text >}}
+
+Use `topology.kubernetes.io/zone` to label each worker with a zone name:
```

Comment:
Maybe make the tip a little shorter?

```suggestion
In a single multi-zone cluster environment, locality load balancing can also be configured for failover to a different zone within the same cluster.
To test it, you will need to create a cluster with multiple worker zones and deploy an istiod instance and the app to each zone.

1) If you don’t have a multi-zone Kubernetes cluster, you can deploy one locally using `kind` with the following command:

{{< text bash >}}
$ kind create cluster --config=- <<EOF
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
  - role: control-plane
  - role: worker
  - role: worker
  - role: worker
EOF
{{< /text >}}

2) Use `topology.kubernetes.io/zone` to label each worker with a zone name:
```

Reply:
Thank you, @frankbu. I committed your suggestion.